js/components/SignupForm/SignupForm.js

```diff
@@ -42,10 +42,14 @@ export default class SignupForm extends Component {
   }
 
   render() {
+    const { navigation } = this.props
     return (
       <ScrollView>
         <Form
-          onSubmit={values => this.onSubmit(values)}
+          onSubmit={values => {
+            this.onSubmit(values)
+            navigation.navigate('Signin')
+          }}
           validate={values => this.validate(values)}
           render={({ handleSubmit, pristine, invalid }) => (
```

Comment:
use pristine to show hide your form depending on form validation.

Reply:
Also, your user should be navigated to a different screen after signup. Is there a logic for that? Please let me know.

Reply:
designers wanted user to be sent back to signin screen once they signup for an account.